roles/confluent.test/molecule/mtls-customcerts-rhel/molecule.yml

```diff
@@ -55,7 +55,7 @@ platforms:
     groups:
       - schema_registry
     image: geerlingguy/docker-centos7-ansible
-    dockerfile: ../Dockerfile.j2
+    # dockerfile: ../Dockerfile.j2
```

Comment:
Please remove this line completely, it's not necessary if we don't use the docker file.